src/components/shared/ManagedTree.css

```diff
@@ -12,6 +12,9 @@
   white-space: nowrap;
   overflow: auto;
   min-width: 100%;
+
+  display: grid;
+  align-content: start;
```

Comment:
what does this do?

Reply:
this will fix bug https://github.com/devtools-html/debugger.html/issues/5149, basically, this will extend items in the tree to full width. I don't know why, but `width: 100%` or `display: flex` was not able to fix the issue.

Reply:
On the current update of master the line already seems to be the full width, so not sure if this is needed.

Reply:
The problem appears when you have long and short short names on the list, and try to vertically scroll, as you can see in the gif.
![devtools-source](https://user-images.githubusercontent.com/7465851/35217634-469fc8c4-ff6c-11e7-9e2e-96f859f4f3f8.gif)